serverless.yml

```diff
@@ -19,7 +18,7 @@ provider:
       Resource: ${env:LOGS_ARN}
     - Effect: Allow
       Action:
-        - Allow: sns:Publish
+        - sns:Publish
```

Comment:
This fixes a serverless configuration warning, but I don't know what the consequences are of it.